packages/devtools_app_shared/lib/src/ui/common.dart

```diff
@@ -600,3 +600,39 @@ class RoundedCornerOptions {
   final bool showBottomLeft;
   final bool showBottomRight;
 }
+
+/// A rounded label containing [labelText].
+class RoundedLabel extends StatelessWidget {
+  const RoundedLabel({
```

Comment:
can we share code with the rounded labels that are used in the widget properties view of the inspector?
![Screenshot 2024-11-21 at 12 54 42 PM](https://github.com/user-attachments/assets/5a37bc5f-468e-4a7b-aedf-77444db71182)